torch/_inductor/kernel/mm.py

```diff
@@ -1315,6 +1315,39 @@ def _to_dtype(x):
         # The future will be awaited at scheduling time in select_algorithm.py
         best_config_future = gen_best_config(mat1, mat2)
 
+    # If not using benchmark_epilogue_fusion, try to inline subgraph for fusion
+    if (
+        not inductor_config.benchmark_epilogue_fusion
+        and inductor_config.enable_inline_subgraph_fusion
+    ):
+        result, winning_choice = autotune_select_algorithm(
+            name,
```

Comment:
This interferes with async compilation. if you were to turn this on a do a benchmark run with  max-autotune, youd see a huge regresssion.